Alloy/commands/compile/index.js

```diff
@@ -1146,7 +1146,7 @@ function optimizeCompiledCode(alloyConfig, paths) {
 		});
 
 		_.each(exceptions.slice(0), function(ex) {
-			exceptions.push(path.join(titaniumFolder, ex));
+			exceptions.push(`${titaniumFolder}/${ex}`);
```

Comment:
`path.posix.join(titaniumFolder, ex)`?

Reply:
Not gonna block on this, looking at it in the whole context we should be safe to do string concat here for path building